backend/services/controlnet_service.py

```
@@ -14,6 +14,7 @@
 import logging
 import httpx
 import base64
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)